CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ##### Enhancements
 
+* Adds ui_test_bundle extension.  
```

Comment:
this should be more specific, saying that we treat `ui_test_bundle` products as `xctest` bundles